src/amber/cli/templates/scaffold/controller/crecto/spec/controllers/{{name}}_controller_spec.cr.ecr

```diff
@@ -36,18 +36,21 @@ class <%= class_name %>ControllerTest < GarnetSpec::Controller::Test
 end
 
 describe <%= class_name %>ControllerTest do
+
+  Spec.before_each do
+    Repo.delete_all(<%= class_name %>)
+  end
+
```

Comment:
This is probably not going to do what you expect. 

> Instructs the spec runner to execute the given block before each spec, regardless of where this method is invoked.
https://crystal-lang.org/api/0.24.2/Spec.html#before_each%28%26block%29-class-method

Due to this weakness with how Spec#before_each is implemented, it should be avoided in generated code at almost all costs.

Reply:
I understand this weakness!